fe/fe-core/src/main/java/com/starrocks/catalog/EsTable.java

```diff
@@ -86,6 +88,15 @@ public class EsTable extends Table {
     public static final String KEY_ES_NET_SSL = "es.net.ssl";
     public static final String KEY_TIME_ZONE = "time_zone";
 
+    // tableContext is used for being convenient to persist some configuration parameters uniformly
+    @SerializedName(value = "tc")
+    private Map<String, String> tableContext = new HashMap<>();
+
+    // only save the partition definition, save the partition key,
+    // partition list is got from es cluster dynamically and is saved in esTableState
+    @SerializedName(value = "p")
+    private PartitionInfo partitionInfo;
```

Comment:
need to register PartitionInfo and its subclasses in GSONUtils?

Reply:
PartitionInfo has register subtype in GSONUtils